tracker/templates/operations.json.j2

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "index-append",
+    "operation-type": "bulk",
+    "bulk-size": {{bulk_size | default(5000)}},
+    "ingest-percentage": {{ingest_percentage | default(100)}}
+},
+{
+    "name": "index-update",
```

Comment:
Unused?

Reply:
I believe so, removed in https://github.com/elastic/rally/pull/875/commits/ff4283398bdce2db2d7eb81158d6f39942347d2f